docs/environment.yaml

```diff
@@ -0,0 +1,19 @@
+channels:
+  - defaults
+  - anaconda
+  - pytorch
+  - conda-forge
+dependencies:
+  - grpcio
+  - grpcio-tools
+  - numpy>=1.14
+  - pytorch>=1.3.0
+  - joblib>=0.13
+  - pip
+  - pip:
+    - recommonmark
+    - sphinx_rtd_theme
+    - prefetch_generator>=1.0.1
+    - uvloop>=0.14.0
+    - umsgpack
+
```

Comment:
note: grpcio here is NOT 1.31, but it is enough of a mockup for docs to build correctly